Gems/GenAIFramework/Code/Include/GenAIFramework/Communication/AIModelRequestBus.h

```diff
@@ -34,10 +36,10 @@ namespace GenAIFramework
     using ModelAPIExtractedResponse = AZ::Outcome<AIMessage, AZStd::string>; //!< The type of extracted response from the model
     using AIHistory = AIMessages; //!< The type of AI history
 
-    class AIModelRequest : public AZ::ComponentBus
+    class AIModelRequests : public AZ::ComponentBus
     {
     public:
-        AZ_RTTI(AIModelRequest, "{8c140e10-e3a2-478b-9886-c7c24eb16816}", AZ::ComponentBus);
+        AZ_RTTI(AIModelRequests, AIModelRequestTypeId, AZ::ComponentBus);
```

Comment:
```suggestion
        AZ_RTTI(AIModelRequests, AIModelRequestsTypeId, AZ::ComponentBus);
```
Nit: changing the typeid name to be more consistent with AIModelRequests change